torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4111,8 +4111,26 @@ def shape(size, rank, with_batch_channel=True):
             return tuple([N, C] + ([size] * rank))
         return tuple([size] * rank)
 
-    make_arg = partial(make_tensor, device=device, dtype=dtype,
-                       requires_grad=requires_grad, low=-1, high=1)
+    if mode in ('bilinear', 'bicubic'):
+        make_arg = partial(
+            make_tensor,
+            device=device,
+            dtype=dtype,
+            requires_grad=requires_grad,
+            low=0 if dtype == torch.uint8 else None,
+            high=256 if dtype == torch.uint8 else None,
```

Comment:
Why are we requiring specific bounds for uint8? By default it will be `[0, 10)`:

https://github.com/pytorch/pytorch/blob/567b5e5b284c40b203fcb9394a2121e0be9b2ae9/torch/testing/_creation.py#L52

Reply:
typical data with uint8 has [0, 255] range and not [0, 10]. This range can effect the error value for upsampling ops, thus I'm picking more representative sample instead of current one

Reply:
Maybe, int8 can be extended to [-128, 128], for other dtypes it is less evident to set natural/typical ranges

Reply:
I'm ok with either. Can we maybe remove the condition for `low` since `0` is the default for uint8 anyway? Plus, it would be good to add a small comment why we are not going with the default `high` value for uint8.